tests/unit/kvmagent/test_nfs_plugin_handlers.py

```
@@ -0,0 +1,1074 @@
+_ = sys.modules.setdefault("traceable_shell", MagicMock())
+_ = sys.modules.setdefault("report", MagicMock())
+_ = sys.modules.setdefault("linux", MagicMock())
+_ = sys.modules.setdefault("bash", MagicMock())
+_ = sys.modules.setdefault("shell", MagicMock())
+
+try:
+    nfs_primarystorage_plugin = cast(
+        _NfsPluginModule,
+        cast(object, importlib.import_module("kvmagent.plugins.nfs_primarystorage_plugin")),
+    )
+except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)